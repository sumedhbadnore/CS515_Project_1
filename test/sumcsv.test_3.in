Item,Quantity,Amount
Apples,5,1.25
Bananas,-3,0.75
Oranges,2,-1.50
Grapes,10,2.25