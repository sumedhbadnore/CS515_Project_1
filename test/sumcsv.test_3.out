Sum Of Column 'Quantity': 14.0